### PR TITLE
Minor changes PT-BR

### PR DESCRIPTION
**Pull request by bitcoinuser**

### CODE REVIEW

File: src/locales/pt_br/translation.json [hunk 1]
```diff
@@ -630,7 +630,7 @@
     "transfer": "Transferir",
     "transferNote": "Por favor, transfira a liquidez que você forneceu para nossas novas pools",
     "v1": {
-      "balance": "Saldo disponível"
+      "balance": "Meu saldo"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:633]:
```suggestion
      "balance": "Saldo disponível"
```

File: src/locales/pt_br/translation.json [hunk 2]
```diff
@@ -1381,7 +1381,7 @@
         "asset": "Colateral:",
         "leverage": "Alavancagem:",
         "amount": "Quantidade:",
-        "balance": "Saldo disponível: <0/> <1/>",
+        "balance": "Meu saldo: <0/> <1/>",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1384]:
```suggestion
        "balance": "Saldo disponível: <0/> <1/>",
```

File: src/locales/pt_br/translation.json [hunk 3]
```diff
@@ -1738,7 +1738,7 @@
     "form": {
       "title": "Comprar SOV com <0>RBTC</0>",
       "enterAmount": "Insira a quantidade",
-      "availableBalance": "Saldo disponível:",
+      "availableBalance": "Meu saldo:",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1741]:
```suggestion
      "availableBalance": "Saldo disponível:",
```

File: src/locales/pt_br/translation.json [hunk 4]
```diff
@@ -405,7 +405,7 @@
     "short": "Vendido"
   },
   "assetWalletBalance": {
-    "accountBalance": "Saldo disponível",
+    "accountBalance": "Meu saldo",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:408]:
```suggestion
    "accountBalance": "Saldo disponível",
```

File: src/locales/pt_br/translation.json [hunk 5]
```diff
@@ -13,7 +13,7 @@
     "failed": "Falhou",
     "deposit": "Depositar",
     "withdraw": "Sacar",
-    "availableBalance": "Saldo disponível:",
+    "availableBalance": "Meu saldo:",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:16]:
```suggestion
    "availableBalance": "Saldo disponível:",
```

**Review by soulBit — CHANGES_REQUESTED** [review]
@bitcoinuser can you remove the duplicate 'title' property from 'stakingForm' please
image

**Comment by bitcoinuser** [comment]
> @bitcoinuser can you remove the duplicate 'title' property from 'stakingForm' please
> image

Done. Can be merged now.

**Pull request merged**